### PR TITLE
Update CHANGELOG.md for v1.1.7 release

### PR DESCRIPTION
**Pull request by roger2hk**
### Checklist

- [x] I have updated the [CHANGELOG](CHANGELOG.md).
  - Adjust the draft version number according to [semantic versioning](https://semver.org/) rules.
- [ ] I have updated [documentation](docs/) accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,31 +2,143 @@
 
 ## HEAD
 
+## v1.1.7
+
+* Recommended go version for development: 1.20
```
Comment (AlCutter) [inline CHANGELOG.md:7]: /go/Go

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,31 +2,143 @@
...
+  * This is the version used by the cloudbuild presubmits. Using a different version can lead to presubmits failing due to unexpected diffs.
```
Comment (AlCutter) [inline CHANGELOG.md:8]: cloudbuild/CloudBuild

**Review by AlCutter — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (roger2hk) [reply CHANGELOG.md]: Done, thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (roger2hk) [reply CHANGELOG.md]: Done, thanks!

**Pull request merged**